create-hpc-cluster/metadata.json

```diff
@@ -2,6 +2,6 @@
   "itemDisplayName": "Create one HPC cluster",
   "description": "This template creates one all in one HPC cluster, head node is acted as domain controller, database is also hosted on head node",
   "summary": "This template creates one allinone HPC cluster",
-  "githubUsername": "justintian",
-  "dateUpdated": "2015-04-07"
+  "githubUsername": "sunbinzhu",
```

Comment:
This is the username of the original author. No need to change this. Github already picks up the contributors to this file.